etc/base.yaml

```diff
@@ -178,17 +178,40 @@ postgresql:
         - name: POSTGRES_MULTIPLE_DATABASES
           value: restsourceauthorizer,appconfig,kratos # ... and these databases will be created
 
+cloudnative_postgresql:
+  _install: true
+  _chart_version: 0.1.0
+  _extra_timeout: 0
+  needs:
+    - cloudnative-pg
```

Comment:
I'm not sure if this should be here.

Reply:
I have tried deploying the cnpg-operator as a helm dependency. This fails because of order problems with the CRD installation. I have now reworked this by 1) changing the creation condition of the operator and 2) keep the `needs:` entry, all the helmfile level, 3) remove the separate _05-..._ helmfile. It cannot get better than this I am afraid.